docs/docs/reference/contextual/givens.md

```diff
@@ -57,14 +57,16 @@ given global as ExecutionContext = new ForkJoinPool()
 This creates a given `global` of type `ExecutionContext` that resolves to the right
 hand side `new ForkJoinPool()`.
 The first time `global` is accessed, a new `ForkJoinPool` is created, which is then
-returned for this and all subsequent accesses to `global`.
+returned for this and all subsequent accesses to `global`. This operation is thread-safe.
 
-Alias givens can be anonymous, e.g.
+Alias givens can be anonymous as well, e.g.
 ```scala
 given Position = enclosingTree.position
 given (using outer: Context) as Context = outer.withOwner(currentOwner)
 ```
-An alias given can have type parameters and implicit parameters just like any other given,
+<!-- TODO IMO the above example is very bad as the outer Context will always have precedence over the anonymous one. No? -->
```

Comment:
This definition is intentional, but I agree that this is a confusing example which should be replaced by something which uses different types.

Reply:
I came up with a random example